### PR TITLE
Remote job submission: add logging

### PR DESCRIPTION
**Pull request by davidpanderson**
Remote job submission requests and errors are now logged to log_host/submit_log.txt.

### CODE REVIEW

**Comment by davidpanderson** [comment]
can someone please merge this?

File: html/inc/submit_util.inc [hunk 1]
```diff
@@ -21,6 +21,26 @@
...
+    global $verbose, $log_file;
+
+    if (!$verbose) return;
+    if (!$log_file) {
+        $hostname = gethostname();
+        $log_dir = "../../log_$hostname";
+        if (!file_exists($log_dir)) {
+            mkdir($log_dir);
+        }
+        $log_file = fopen("$log_dir/submit_log.txt", "a");
```
Comment (lfield) [inline html/inc/submit_util.inc]: This fails with `fopen(../../log_boincai05.cern.ch/submit_log.txt): failed to open stream: No such file or directory in <b>/boincdata/boinc/project/lhcathome-dev/html/inc/submit_util.inc`

Note that the existing directory is log_boincai05 so $hostname should not contain cern.ch

Comment (lfield) [reply html/inc/submit_util.inc]: The issue is that the log dir should come from the configuration. Elsewhere it uses
parse_config(get_config(), "<log_dir>");

Comment (lfield) [reply html/inc/submit_util.inc]: To be consistent the file should be called submit.log rather than submit_log.txt.

**Comment by lfield** [comment]
Please could you briefly explain the difference to the remote_submission.log?

**Comment by davidpanderson** [comment]
I fixed all the issues.

**Comment by lfield** [comment]
Tested and it looks good.

**Pull request merged**